sale_order_lot_selection/tests/test_sale_order_lot_selection.py

```diff
@@ -0,0 +1,321 @@
+# © 2015 Agile Business Group
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo.tests import TransactionCase
```

Comment:
```suggestion
from odoo.addons.base.tests.common import BaseCommon
```
It is recommended to use BaseCommon.